modules/dataframe/src/io/csvreader.cpp

```diff
@@ -586,6 +582,7 @@ std::shared_ptr<DataFrame> CSVReader::readData(std::istream& stream) const {
     }
 
     std::string content{std::istreambuf_iterator<char>(stream), std::istreambuf_iterator<char>()};
+    content = rtrim(content);
```

Comment:
ahh don't use that function. you should return a string_view not a copy of the content.